readthedocs/rtd_tests/fixtures/spec/v2/schema.json

```diff
@@ -0,0 +1,330 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "$comment": "https://docs.readthedocs.io/en/stable/development/design/yaml-file.html",
```

Comment:
We should link to https://docs.readthedocs.io/en/stable/config-file/index.html instead, the design doc is outdated now.